### PR TITLE
Custom maxwait on cleanup jobs

### PR DESCRIPTION
**Pull request by anastasiamac**
## Description of change

Clean up jobs that can be scheduled now have the ability to wait a certain period of time. For now, it's either system default of 1 Minute or user defined ) seconds (don't wait at all).

### CODE REVIEW

**Comment by anastasiamac** [comment]
[wire up max wait to scheduling jobs](https://trello.com/c/4lc0ck1K/710-wire-up-max-wait-to-scheduling-jobs)

File: apiserver/common/remove.go [hunk 1]
```diff
@@ -85,3 +86,12 @@ func (r *Remover) Remove(args params.Entities) (params.ErrorResults, error) {
 	}
 	return result, nil
 }
+
+// MaxWait is how far in the future the backstop force cleanup will be scheduled.
+// Default is 1min if no value is provided.
+func MaxWait(in *time.Duration) time.Duration {
```
Comment (babbageclunk) [inline apiserver/common/remove.go]: This is really nice.

**Comment by anastasiamac** [comment]
$$merge$$

**Comment by anastasiamac** [comment]
$$merge$$

**Comment by anastasiamac** [comment]
$$merge$$

**Pull request merged**